### PR TITLE
Behavior in storage setup for metrics and logging not properly documented and inconsistent

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1732804

### CODE REVIEW

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -304,13 +304,15 @@ appended to the prefix starting from 1.
 |The persistent volume claim size for each of the Cassandra nodes.
 
 |`openshift_metrics_cassandra_pvc_storage_class_name`
-|If you want to explicitly set the storage class, you must not set
-`openshift_metrics_cassandra_storage_type` to `emptydir` or `dynamic`.
+|Specify the storage class to use. If you want to explicitly set the storage class, 
+use `openshift_logging_elasticsearch_storage_type=pv`. This parameter does not work
```
Comment (anpingli) [inline install_config/cluster_metrics.adoc:308]: @mburke5678  Do you ream openshift_metrics_cassandra_storage_type=pv here?

Comment (mburke5678) [reply install_config/cluster_metrics.adoc:308]: @anpingli 
Yes. openshift_metrics_cassandra_storage_type=pv  Thank you!

**Comment by mburke5678** [comment]
@juzhao PTAL

File: install_config/cluster_metrics.adoc [hunk 2]
```diff
@@ -304,13 +304,15 @@ appended to the prefix starting from 1.
 |The persistent volume claim size for each of the Cassandra nodes.
 
 |`openshift_metrics_cassandra_pvc_storage_class_name`
-|If you want to explicitly set the storage class, you must not set
-`openshift_metrics_cassandra_storage_type` to `emptydir` or `dynamic`.
+|Specify the storage class to use. If you want to explicitly set the storage class, 
+use `openshift_metrics_cassandra_storage_type=pv`. This parameter does not work
```
Comment (juzhao) [inline install_config/cluster_metrics.adoc:308]: description for openshift_metrics_cassandra_pvc_storage_class_name
Specify the storage class to use. If you want to explicitly set the storage class, use openshift_metrics_cassandra_storage_type=pv
-> use openshift_metrics_cassandra_storage_type=pv
I think it is better change to:
should also set openshift_metrics_cassandra_storage_type=pv

File: install_config/cluster_metrics.adoc [hunk 3]
```diff
@@ -304,13 +304,15 @@ appended to the prefix starting from 1.
 |The persistent volume claim size for each of the Cassandra nodes.
 
 |`openshift_metrics_cassandra_pvc_storage_class_name`
-|If you want to explicitly set the storage class, you must not set
-`openshift_metrics_cassandra_storage_type` to `emptydir` or `dynamic`.
+|Specify the storage class to use. If you want to explicitly set the storage class, 
+use `openshift_metrics_cassandra_storage_type=pv`. This parameter does not work 
+if `openshift_metrics_cassandra_storage_type` is `emptydir` or `dynamic`.
```
Comment (juzhao) [inline install_config/cluster_metrics.adoc:309]: Tested on a clouldprovider enabled 311 cluster, openshift_metrics_cassandra_storage_type  is emptydir or dynamic can work with openshift_metrics_cassandra_pvc_storage_class_name
example:
***************
openshift_metrics_cassandra_storage_type=emptydir(if set to emptydir, no PV is created)
openshift_metrics_cassandra_pvc_storage_class_name=standard
***************
openshift_metrics_cassandra_storage_type=dynamic
openshift_metrics_cassandra_pvc_storage_class_name=standard
***************

**Comment by mburke5678** [comment]
@juzhao Changed to: 
```
|`openshift_metrics_cassandra_pvc_storage_class_name`
|Specify the storage class to use. If you want to explicitly set the storage class, 
also set `openshift_metrics_cassandra_storage_type=pv`. 
```

**Pull request merged**